### PR TITLE
Topic/app 1/remove patient checkin componant

### PR DESCRIPTION
**Pull request by Liriaene**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

**Review by ElijahRW — CHANGES_REQUESTED** [review]
This PR doesn't compile: 
* Please properly remove the components from the places they've been referenced (such as `app.module.ts`) before disconnecting them.
``` 
./src/app/app.routes.ts
Module not found: Error: Can't resolve './check-in/check-in.component' in 'C:\Users\elija\Desktop\capstone_app\EPRW_Work\VA-Audiology-App\src\app'
 @ ./src/app/app.routes.ts 3:0-65
 @ ./src/app/app.module.ts
 @ ./src/app/index.ts
 @ ./src/main.browser.ts
 @ multi (webpack)-dev-server/client?http://localhost:3000 ./src/main.browser.ts
errors @ client:139
```

File: src/app/home/home.component.html [hunk 1]
```diff
@@ -33,14 +33,7 @@
       <div class="row">
         <img src= "assets/images/homescreenimage.png" id="home-img">
       </div>
-
-      <div class="row">
-         <div class="col-sm-8 col-md-6 col-md-offset-3 col-sm-offset-2">
-            <button (click)="patientOnClick()" type="button" class = "btn btn-success">Patient Check In</button>
-        </div>
```
Comment (OkapalDominic) [inline src/app/home/home.component.html]: Why was this removed?  PR #69 will change where it routes to.  Maybe wait until that one goes through, and then rebase.

Comment (Liriaene) [reply src/app/home/home.component.html]: There wasn't a real reason I removed it. I figured since the component was ripped out the button shouldn't be there either. I can add it back in.

**Review by OkapalDominic — APPROVED** [review]

**Comment by gstone271** [comment]
The git history here is a mess, you probably want to just start over with a new branch on an updated version of development (make sure to pull development). 

The checkin component seems to be still present in the code even though it is unused? It should be removed.

Completely unrelated Routes code seems to have been added? Where did this code even come from?

**Comment by gstone271** [comment]
Duplicate of #96 ?

**Comment by Liriaene** [comment]
Moved to ticket #96
